guides/models.md

```diff
@@ -7,6 +7,9 @@ The following is a base template for an ActiveRecord model `Customer`.
 * Avoid ActiveRecord callbacks where possible, prefer Service Objects architecture to store any business
 * Order associations, scopes and validations alphabetically
 * Aim to remove business logic that is not directly related to reading from or writing to the database
+* `Scopes` vs `Class methods`. Use scopes when the logic is very small, for simple where/order clauses,
```

Comment:
We have `Query Objects` for complex queries, sqls, and "scopes" - https://github.com/ergoserv/handbook/blob/master/guides/query_objects.md.
I'd suggest to use them instead of `Class methods`. What do you think?